app/src/main/java/com/hedvig/app/util/apollo/SafeQuery.kt

```diff
@@ -50,6 +50,7 @@ fun <D : Subscription.Data> ApolloCall<D>.safeSubscription(): Flow<QueryResult<D
 
 fun <D : Query.Data> ApolloCall<D>.safeWatch(): Flow<QueryResult<D>> {
     return try {
+        watch(null)
```

Comment:
What's going on here?

Reply:
A mistake is going on here 🙊 I thought I removed it but alas I had not.
I was playing around with the stuff I was looking through in [this](https://kotlinlang.slack.com/archives/C01A6KM1SBZ/p1652857651123899?thread_ts=1652794902.304599&cid=C01A6KM1SBZ) discussion and forgot to remove it :D